modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -261,8 +261,6 @@ export default {
       await this.setContext({
         doc: layer.doc
       });
```

Comment:
In this case, the refresh is performed from the `setContext` method when `contextStack` is empty. This is the case in the `onPopContext` method.